content/20.treatments.md

```diff
@@ -198,16 +198,63 @@ Summarize the state of the antiviral approach.
+The passive transfer of immune serum containing nAbs from SARS-CoV-infected mice resulted in protection of naïve mice from viral lower respiratory tract infection upon intranasal challenge [@doi:10.1128/jvi.78.7.3572-3577.2004].
+Similarly, a meta-analysis suggested that administration of plasma from recovered SARS-CoV patients reduced mortality upon SARS-CoV infection [@doi:10.1093/infdis/jiu396]. 
+
+Similar results have been observed for MERS-CoV infections, which emerged as the second coronavirus-related epidemic.
+Neutralizing antibodies have been identified against various epitopes of the RBD of the S glycoprotein [@doi:10.1073/pnas.1402074111; doi:10.1128/JVI.00912-14].
+
+###### Spike (S) Neutralizing Antibody Anticipated Mechanisms
+
+Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to host cell receptors, such as ACE2, allowing for cell entry [@doi:10.1016/j.cell.2020.02.052; doi:10.1016/j.cell.2020.02.058].
+Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
+The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755].
+Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection [@doi:10.1038/nature02145], given the physiological relevance of ACE2 [@doi:10/bsbp49], it would be favorable to target virus-specific structures rather than host receptors.
+This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.
+
+
+###### Spike (S) Neutralizing Antibody Current Evidence
+
+The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology, [@doi:10.1101/2020.03.11.987958], where antibody-producing B-cells developed by mice can be inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
+The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
+The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure [@doi:10.1101/2020.03.11.987958; @doi:10.1016/j.cell.2018.12.028].
+The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy [@doi:10.1128/JVI.01603-17].
+Only a week later, a different group reported the isolation of multiple nAbs targeting the RBD of the S glycoprotein from COVID-19 patients [@doi:10.1101/2020.03.21.990770].
```

Comment:
```suggestion
Only a week later, a different group successfully isolated multiple nAbs targeting the RBD of the S glycoprotein from blood samples taken from COVID-19 patients in China [@doi:10.1101/2020.03.21.990770].
```
That totally makes sense and is obvious now that you say it! I think I had trouble noticing that the last bit of the sentence was the main emphasis because my brain was preoccupied by remembering what the RBD of the S glycoprotein is. Here's an idea but it's definitely not a critical change -- I just added a little more info in the hopes it might help the second half of the sentence more salient!